### ISSUE TITLE
ui: browser support

### ISSUE THREAD

**Issue by zven** [mint-o-badges/badgr-ui#437]

**Comment by zven** [comment]
@Annicelia Our team mainly uses Firefox, Chrome and Safari. Microsoft Edge has not been tested at all during development so far. Do we need to address this?

**Comment by scheidtdav** [comment]
Yep correct, there should barely be a difference if at all. I suppose that was even one of the reasons why Microsoft chose to go with Chromium in the first place.

Firefox, Chrome (or Edge) and Safari (or GNOME Web/ Epiphany) cover the three big players (98%+ market share) on the market when it comes to browser engines: Gecko, Blink and WebKit.
